nss-day-cohort-48/giffygram-webcrawlers
# Dmg new post

#### Changes Made
1. Edited giffygram.js to return a properly formatted string
2. Added some margin to the top of the page so that the navbar is not covering the post form
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout dmg-new-post
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    ensure that full page renders and that you can see the gif post button and form
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
